File: Ruby/1012.rb
a, b, c = gets.split
a = a.to_f
b = b.to_f
c = c.to_f

tri = a*c/2
cir = 3.14159*(c**2)
tra = (a+b)*c/2
qua = b*b
ret = a*b

puts sprintf("TRIANGULO: " + "%.3f", tri)
puts sprintf("CIRCULO: " + "%.3f", cir)
puts sprintf("TRAPEZIO: " + "%.3f", tra)
puts sprintf("QUADRADO: " + "%.3f", qua)
puts sprintf("RETANGULO: " + "%.3f", ret)

File: Python/1012.py
A, B, C = input().split()

A = float(A)
B = float(B)
C = float(C)

print('TRIANGULO: {:.3f}'.format((A*C)/2))
print('CIRCULO: {:.3f}'.format(3.14159*C**2))
print('TRAPEZIO: {:.3f}'.format(((A+B)*C)/2))
print('QUADRADO: {:.3f}'.format(B**2))
print('RETANGULO: {:.3f}'.format(A*B))